.gitignore

```diff
@@ -2,6 +2,8 @@
 ### GitHub repository .gitignore section
 ###
 
+composer.lock
```

Comment:
Yes. I meant to do that for a few different reasons. Mainly so that there weren't any merge conflicts like there have been sometimes lately. And two just thought it would be easier for Circle to grab the most recent version by default rather then grabbing things from the lock file.